src/server/game/Spells/SpellEffects.cpp

```diff
@@ -2137,8 +2137,10 @@ void Spell::EffectOpenLock(SpellEffIndex effIndex)
         else if (m_spellInfo->Id == 1842 && gameObjTarget->GetGOInfo()->type == GAMEOBJECT_TYPE_TRAP)
         {
             gameObjTarget->SetLootState(GO_JUST_DEACTIVATED);
-            if (!gameObjTarget->GetOwner()) // pussywizard
+            if (gameObjTarget->getLootState() == GO_JUST_DEACTIVATED && !gameObjTarget->GetOwner()) // pussywizard
```

Comment:
pussywizard? is this a partial revert? or from another project?

Reply:
I think the comment was there all the time

Reply:
Ah gotcha. I must of overlooked it entirely.